sklearn/linear_model/ridge.py

```diff
@@ -1484,7 +1484,7 @@ def identity_estimator():
             identity_estimator.decision_function = lambda y_predict: y_predict
             identity_estimator.predict = lambda y_predict: y_predict
 
-            out = [scorer(identity_estimator, y.ravel(), cv_values[:, i])
+            out = [scorer(identity_estimator, cv_values[:, i], y.ravel())
```

Comment:
scorers have first y_true and then y_pred in their signature. See eg : https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.r2_score.html#sklearn.metrics.r2_score

are you sure it's correct?

Reply:
I think that I have explained this in the refereed issue. 

To summarize  :  

`scorer` is not a metric. It is a scoring function,  coming out of [`check_scoring()`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.check_scoring.html)  [here](https://github.com/scikit-learn/scikit-learn/blob/1495f6924/sklearn/linear_model/ridge.py#L1429).

The signature of scorer is 
`scorer(estimator, X, y)`

Reply:
Add a comment please.

Reply:
Add a comment in the code to help the reader